misc/toml-parser/src/main/java/io/ballerina/toml/validator/SchemaValidator.java

```diff
@@ -66,12 +68,11 @@ public SchemaValidator(Schema schema) {
     public void visit(TomlTableNode tomlTableNode) {
         if (schema.type() != Type.OBJECT) {
             TomlDiagnostic diagnostic = getTomlDiagnostic(tomlTableNode.location(), "TVE0002", "error.invalid.type",
```

Comment:
Shall we define constants for `"TVE0002"` and `"error.invalid.type"`?

Reply:
This should be addressed with templates with all the error messages. We've created an issue to keep track of this. #27041
Shall we do this change with that PR?